src/dynafed.rs

```diff
@@ -272,3 +369,81 @@ impl Decodable for Params {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    use bitcoin;
+    use bitcoin::hashes::hex::ToHex;
+
+    #[test]
+    fn test_param_roots() {
+        // Taken from the following Elements Core test:
```

Comment:
Would have been better I think to just include a github link to commit and line number. But not all that important.

Reply:
Yeah the test in Elements was still up for review and might not be merged or be changed in the future. I figured it's not that big so better just make sure to have it.